sdk/storage/Azure.Storage.Files.Shares/src/Models/ShareFileItemProperties.cs

```diff
@@ -0,0 +1,45 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+
+namespace Azure.Storage.Files.Shares.Models
+{
+    /// <summary>
+    /// Properties of a <see cref="ShareFileItem"/>.
+    /// </summary>
+    public class ShareFileItemProperties
+    {
+        internal ShareFileItemProperties() { }
```

Comment:
Can we make a dedicated internal constructor that takes in all the properties?

Reply:
+1

Reply:
done.